src/scene/scene.js

```diff
@@ -65,6 +68,7 @@ import { StandardMaterial } from './materials/standard-material.js';
  * @property {pc.Texture} skyboxPrefiltered8 The prefiltered cubemap texture (size 8x8) used as the scene's skybox, if mip level 5. Defaults to null.
  * @property {pc.Texture} skyboxPrefiltered4 The prefiltered cubemap texture (size 4x4) used as the scene's skybox, if mip level 6. Defaults to null.
  * @property {number} skyboxIntensity Multiplier for skybox intensity. Defaults to 1.
+ * @property {pc.Quat} skyboxRotation The rotation of the skybox to be displayed. Defaults to pc.Quat.IDENTITY
```

Comment:
Pedantic alert! 🚨  All other @property lines end with a full stop. 😄

Reply:
Even better would be:

```
Defaults to {@link pc.Quat.IDENTITY}.
```

Reply:
That is better - fixed